### ISSUE TITLE
c# compiled exe load supported ?

### ISSUE THREAD

**Issue by noname29** [fancycode/MemoryModule#107]
Can you load an executable that written in c# in this memorymodule ?  I want to use this memoryModule specifically if so please help me on what would be needed.. Any problem with mscoree.dll ? i don't understand. Thank you so much..

**Comment by Sardelka9515** [comment]
No, .NET codes are compiled to MSIL. The library only supports loading native win32 modules.
